src/pages/donors.js

```diff
@@ -1,3 +1,41 @@
+import Donor from '../components/Donor';
+
+import Grid from '../utils/Grid';
+
 export default function Donors() {
-  return <p>Donors</p>;
+  const donorArr = [
+    {
+      id: 1,
+      first_name: 'Jon',
+      last_name: 'Smith',
+    },
+    {
+      id: 2,
+      first_name: 'Matt',
+      last_name: 'Clark',
+    },
+    {
+      id: 3,
+      first_name: 'Mitchell',
+      last_name: 'Pavel',
+    },
+    {
+      id: 4,
+      first_name: 'Bob',
+      last_name: 'Chin',
+    },
+  ];
+
+  const donors = donorArr.map((donor) => <Donor key={donor.id} donor={donor} />);
```

Comment:
Rename `donors` to `ministryPartners`, and `donor` to `ministryPartner`.

Move the map statement into the markup.